mix.exs

```diff
@@ -19,23 +19,10 @@ defmodule Parma.Mixfile do
   #
   # Type `mix help compile.app` for more information.
   def application do
-    [mod: {Parma, []},
-     applications: [
-                    :comeonin,
-                    :phoenix,
-                    :phoenix_pubsub,
-                    :phoenix_html,
-                    :cowboy,
-                    :logger,
-                    :gettext,
-                    :phoenix_ecto,
-                    :postgrex,
-                    :tentacat,
-                    :ueberauth,
-                    :ueberauth_github,
-                    :toniq
-                    ]
-     ]
+    [
+      mod: {Parma, []},
+      extra_applications: [:logger]
```

Comment:
Its not required after Elixir 1.4 because of [Application Inference](https://elixir-lang.org/blog/2017/01/05/elixir-v1-4-0-released/#application-inference)